integration-tests/src/test/java/org/apache/fineract/integrationtests/investor/externalassetowner/InitiateExternalAssetOwnerTransferTest.java

```diff
@@ -933,6 +936,35 @@ public void saleExceptionHandling() {
         }
     }
 
+    @Test
+    public void buybackOnATobeDeclinedPendingSale() {
```

Comment:
Please remove this test case as it is part of an another PR.

Reply:
Done! Test case removed from this class